README.md

```diff
@@ -13,21 +13,33 @@
 [![][license-epl img]][license-epl]
 [![][license-edl img]][license-edl]
 
-![logo](artwork/eclipse-collections-logo.png) 
+<a href="http://www.eclipse.org/collections"><img src="https://github.com/eclipse/eclipse-collections/blob/master/artwork/eclipse-collections-logo.png" height="50%" width="50%"></a>
 
+#### [中文](https://www.eclipse.org/collections/cn/index.html) | [Deutsch](https://www.eclipse.org/collections/de/index.html) | [Français](https://www.eclipse.org/collections/fr/index.html) | [日本語](https://www.eclipse.org/collections/ja/index.html) | [Português-Brasil](https://www.eclipse.org/collections/pt-br/index.html) | [Русский](https://www.eclipse.org/collections/ru/index.html)
+Eclipse Collections is a comprehensive collections library for Java. The library enables productivity and performance by delivering an eloquent, elegant and efficient set of APIs and types. The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
```

Comment:
For the second sentence, what do you think about:
"The library can be used as a large supplement to, or complete replacement for the built-in Java Collection Framework types."

Reply:
In place of or in addition to the second sentence?